### PR TITLE
Accounts V2: Fix issues reported by rkapka

### PR DESCRIPTION
**Pull request by 0xKiwi**
**What type of PR is this?**
Bug fix and cleanup

**What does this PR do? Why is it needed?**
Part of #6723, this solves some of the issues reported by rkapka. Issues includes:
--skip-mnemonic-confirm being on `accounts-v2 create help`
--password-file flag not being allowed in runtime
Allowing "y" or "Y" for prompt confirmation entry.
Cleaning up errors for invalid words entered for mnemonics, or just entirely invalid mnemonics
Fixing formatting of accounts list from derived vs. non-hd
Solves some formatting and test issues for logs 
Adds an error if you try to import non-HD accounts into a HD wallet 

Also fixes a bug with password file entry

### CODE REVIEW

**Comment by rkapka** [comment]
The referenced issue has been merged into #6715.

File: validator/accounts/v2/accounts_list.go [hunk 1]
```diff
@@ -106,7 +104,9 @@ func listDirectKeymanagerAccounts(
 			return errors.Wrapf(err, "could not parse account created at timestamp: %s", createdAtBytes)
 		}
 		unixTimestamp := time.Unix(unixTimestampStr, 0)
-		fmt.Printf("%s %s\n", au.BrightCyan("[created at]").Bold(), humanize.Time(unixTimestamp))
+
+		fmt.Printf("%s | %s\n", au.BrightGreen(accountNames[i]).Bold(), humanize.Time(unixTimestamp))
+		fmt.Printf("%s %#x\n", au.BrightMagenta("[validating public key]").Bold(), pubKeys[i])
```
Comment (0xKiwi) [inline validator/accounts/v2/accounts_list.go:109]: We should abstract this logging into a shared function if possible

File: validator/accounts/v2/wallet_create.go [hunk 2]
```diff
@@ -21,23 +21,26 @@ func CreateWallet(cliCtx *cli.Context) error {
 		return err
 	}
 	w, err := NewWallet(cliCtx, keymanagerKind)
-	if err != nil {
+	if err != nil && !errors.Is(err, ErrWalletExists) {
 		return errors.Wrap(err, "could not check if wallet directory exists")
 	}
+	if errors.Is(err, ErrWalletExists) {
+		return ErrWalletExists
```
Comment (0xKiwi) [inline validator/accounts/v2/wallet_create.go:28]: Just returning the wallet exists error here so it doesn't pass through `could not check if wallet directory exists`

File: validator/flags/flags.go [hunk 3]
```diff
@@ -136,7 +136,7 @@ var (
 	// PasswordFileFlag is used to enter a file to get a password for new account creation, non-interactively.
 	PasswordFileFlag = &cli.StringFlag{
 		Name:  "password-file",
-		Usage: "Path to a file containing a password to interact with wallets/accounts in a non-interactive way",
+		Usage: "Path to a file containing a password in text to interact with wallets/accounts in a non-interactive way",
```
Comment (rauljordan) [inline validator/flags/flags.go:139]:
```suggestion
		Usage: "Path to a plaintext password.txt file",
```

**Pull request merged**